Packs/AWS-Athena/Integrations/AWS-Athena/AWS-Athena.py

```diff
@@ -1,248 +1,299 @@
-            aws_secret_access_key=sts_response['Credentials']['SecretAccessKey'],
-            aws_session_token=sts_response['Credentials']['SessionToken'],
-            verify=VERIFY_CERTIFICATE,
-            config=config
-        )
     else:
-        if region is not None:
-            client = boto3.client(
-                service_name=service,
-                region_name=region,
-                aws_access_key_id=AWS_ACCESS_KEY_ID,
-                aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
-                verify=VERIFY_CERTIFICATE,
-                config=config
-            )
-        else:
-            client = boto3.client(
-                service_name=service,
-                region_name=AWS_DEFAULT_REGION,
-                aws_access_key_id=AWS_ACCESS_KEY_ID,
-                aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
-                verify=VERIFY_CERTIFICATE,
-                config=config
-            )
-
-    return client
-
-
-def start_query_execution_command(args):
-    client = aws_session(
-        region=args.get('region'),
-        roleArn=args.get('roleArn'),
-        roleSessionName=args.get('roleSessionName'),
-        roleSessionDuration=args.get('roleSessionDuration'),
-    )
-    data = []
-    kwargs = {'QueryString': args.get('QueryString')}
-    if args.get('ClientRequestToken') is not None:
-        kwargs.update({'ClientRequestToken': args.get('ClientRequestToken')})
-    if args.get('Database') is not None:
-        kwargs.update({'QueryExecutionContext': {'Database': args.get('Database')}})
-    if args.get('OutputLocation') is not None:
-        kwargs.update({'ResultConfiguration': {'OutputLocation': args.get('OutputLocation')}})
-    if args.get('EncryptionOption') is not None:
-        kwargs.update({'ResultConfiguration': {'EncryptionConfiguration': {'EncryptionOption': args.get('EncryptionOption')}}})
-    if args.get('KmsKey') is not None:
-        kwargs.update({'ResultConfiguration': {'EncryptionConfiguration': {'KmsKey': args.get('KmsKey')}}})
-    if args.get('WorkGroup') is not None:
-        kwargs.update({'WorkGroup': args.get('WorkGroup')})
-
-    response = client.start_query_execution(**kwargs)
-
-    data.append({
-        'QueryString': args.get('QueryString'),
+        raise DemistoException(f'Error: {response}')
+
+
+def start_query_command(args: dict, client):
+    query_string: str = args['QueryString']
+
+    response = start_query_execution(client=client, query_string=query_string, query_limit=args.get('QueryLimit'),
+                                     client_request_token=args.get('ClientRequestToken'), database=args.get('Database'),
+                                     output_location=args.get('OutputLocation'), encryption_option=args.get('EncryptionOption'),
+                                     kms_key=args.get('KmsKey'), work_group=args.get('WorkGroup'))
+
+    context_data = {
+        'QueryString': query_string,
```

Comment:
This is the dictionary I push to context using `outputs_prefix=f'AWS.Athena.{QUERY_DATA_OUTPUTS_KEY}',`.
So the final context path would be `AWS.Athena.Query.QueryString` (this specific item is a string containing the SQL query that was used in the command)

Reply:
@MichaelYochpaz,yes, but the key in the yml are `AWS.Athena.Query.Query` :)

Reply:
You are right, it was changed while I was working on it and I forgot to update it.
Should be `Query` to match the key that's returned by the API when running `aws-athena-get-query-execution` (to avoid redundant keys with different names).
I pushed a fix :)